DemoApp/package.json

```diff
@@ -24,7 +24,8 @@
     "e2e:android": "run-p --race start e2e:test:android",
     "e2e:ios": "run-p --race start e2e:test:ios",
     "e2e:android:micro": "run-s -c micro:run e2e:android micro:stop",
-    "e2e:ios:micro": "run-s -c micro:run e2e:ios micro:stop"
+    "e2e:ios:micro": "run-s -c micro:run e2e:ios micro:stop",
+    "postinstall": "rm -rf node_modules/@snowplow/react-native-tracker/{.git,node_modules,DemoApp,DemoAppTV}"
```

Comment:
I did not know that `yarn` [ignores `files`](https://github.com/yarnpkg/yarn/issues/2822) in package.json for a `file:..` dependency. It does not seem to "install" though, but appears to just copy the tree. 

An alternative could be using `npm pack` (also) in bootstrap script and use the generated `file: ../snowplow-react-native-tracker-1.2.1.tgz` as the tracker dependency in demo apps? What do you think?

If we still go this way, could we use `rimraf` instead of `rm -rf` for an OS/shell independent way?

Reply:
I like the idea to `npm pack`. I tried it out and it worked but was running into issues in CI. Checking the package checksum was failing as yarn was trying to match it to the checksum of the published version (at least that's my understanding). Even when ignoring the checksum check, yarn was producing more errors.

So I decided to go back to the previous approach for now, but used `rimraf` instead of `rm -rf` as you suggested.